CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [4.1.1] - 2022-07-29
```

Comment:
Please remove this line. We will add it later during the release

Reply:
Done!
My bad